src/components/Audiences/injectProcessDestinations.js

```diff
@@ -55,7 +55,6 @@ export default ({
             // We intentionally do not throw an error if destinations fail. We
             // consider it a non-critical failure and therefore do not want it to
             // reject the promise handed back to the customer.
-            logger.error(createResultLogMessage(urlDestination, false));
```

Comment:
maybe we log a warning, or an error only when debug is enabled?

Reply:
An error in debug mode is the current behavior. However, the error message reads "URL destination failed." We don't have a way to know if it failed because the server returned a 400 response code or an empty 200 response code (empty 200 response codes were what prompted this ticket).